src/puppet-js-view-model-tree.html

```diff
@@ -91,6 +91,10 @@
       var puppet = listener.getPuppetClient();
       if (puppet) {
         clone.querySelector('juicy-jsoneditor').json = puppet.obj;
+        // passover changes to Puppet
+        clone.querySelector('juicy-jsoneditor').addEventListener('change', function(e) {
+          puppet.puppet.handleLocalChange(e.detail.patches);
```

Comment:
I'll investigate this tomorrow. Because I guess calling `handleLocalChanges` isn't an option with upcoming proxies, considering the case of adding an object (or array) using `sc-debug-aid`. Calling `handleLocalChange` will allow it to slip unproxified.